temporal/client.go

```diff
@@ -88,10 +87,6 @@ func temporalClientOptions(logger *ZapAdapter, params ProviderParameters) (*clie
 		Interceptors:       interceptors,
 	}
 
-	if config.ClientSideEncryptionEnabled {
-		options.DataConverter = NewEncryptionDataConverter(logger, converter.GetDefaultDataConverter(), EncryptionDataConverterOptions{CMKARNs: config.ClientSideEncryptionCMKARNs})
-	}
```

Comment:
I take it this is because we thought we were going to do client encryption for temporal, never turned it on cuz debugging in temporal would be a pain, then found out it wasn't required?

Reply:
yes exactly, and some of the methods we relied on have been remove from the client